### PR TITLE
Adding 4 docker networking  commands

### PR DESCRIPTION
**Pull request by shubham-kshetre**
Adding docker network commands.

### CODE REVIEW

File: docker [hunk 1]
```diff
@@ -57,3 +57,15 @@ docker login
 
 # To push a docker image into dockerhub repository:
 docker push <image-name>:<image-tag-name>
+
+# To list all networks daemon knows
+docker network ls
```
Comment (EmilySeville7cfg) [inline docker:62]:
```suggestion
# List all networks daemon knows about:
docker network ls
```

- Optional change: `to` is not mandatory.
- Optional change: `a specific` may sound better.
- Optional change: maybe it's better to use a trailing comma like in TlDr pages.

File: docker [hunk 2]
```diff
@@ -57,3 +57,15 @@ docker login
...
+
+# To create a network
+docker network create <network name>
```
Comment (EmilySeville7cfg) [inline docker:65]:
```suggestion
# Create a specific network:
docker network create "<network_name>"
```

- Almost the same as above.
- Optional change: it's safer to use quotes.
- Optional change: it's better to be consistent: use snake-style placeholders.

File: docker [hunk 3]
```diff
@@ -57,3 +57,15 @@ docker login
...
+
+# To connect a container to a network
+docker network connect <NETWORK ID>or<NAME> <CONTAINER ID>or<NAME>
```
Comment (EmilySeville7cfg) [inline docker:68]:
```suggestion
# Connect a specific container to a network:
docker network connect "<network_id|name>" "<container_id|name>"
```

- Almost the same issue as above.
- Optional change: use TlDr page style delimiters inside placeholders.

File: docker [hunk 4]
```diff
@@ -57,3 +57,15 @@ docker login
...
+
+# To disconnect a container from network
+docker network disconnect <NETWORK ID>or<NAME> <CONTAINER ID>or<NAME>
```
Comment (EmilySeville7cfg) [inline docker:71]:
```suggestion
# Disconnect a specific container from network:
docker network disconnect "<network_id|name>" "<container_id|name>"
```

Almost the same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
- Approved because no required changes found.
- Thanks for a contribution. ❤️

**Comment by chrisallenlane** [comment]
Merged cheatsheet and proposed edits. Thanks.

**Pull request merged**